src/Api/Controllers/DevicesController.cs

```diff
@@ -186,6 +186,19 @@ public async Task PutToken(string identifier, [FromBody] DeviceTokenRequestModel
         await _deviceService.SaveAsync(model.ToDevice(device));
     }
 
+    [HttpPut("identifier/{identifier}/web-push-auth")]
+    [HttpPost("identifier/{identifier}/web-push-auth")]
```

Comment:
❓ Why also `POST`?

Reply:
Entirely to match PutToken, which is perfectly analogous to this method but for Firebase/APN.

I don't understand why there's a post there, though...